### ISSUE TITLE
Captchas not solved when iframe not in a form

### ISSUE THREAD

**Issue by tuckerconnelly** [berstend/puppeteer-extra#57]
This line here assumes that recaptcha is in a form: https://github.com/berstend/puppeteer-extra/blob/master/packages/puppeteer-extra-plugin-recaptcha/src/content.ts#L142

But there are certain sites that do not wrap their captcha in a form.

Workaround is to move the captcha to a form on the page before calling the `page.solveRecaptchas()` function:

```js
    await page.evaluate(() => {
      document
        .querySelector('form')
        .append(document.querySelector('#login-recaptcha'));
    });
    await page.solveRecaptchas();
```

Thank you very much for this library, extremely helpful, but my 17 years of experience are telling me the source code is a bit over-engineered. I recommend only using a function or two instead of a class when possible. Nonetheless, thank you so much, this library saved my tush :)

**Comment by berstend** [comment]
Over-engineering is in the eyes of the beholder 😄 Would I write the code differently now (after a year)? Of course, I understand the problem domain much better now. Do I think a function or two would be more maintainable than using TypeScript classes for organizing the code? My 20 years of experience are not so sure about that 😜

But yeah, the code could benefit from some refactoring but ain't no one got time for that. :)

I'll see that I incorporate the form fix in one of the future updates, thanks!

**Comment by berstend** [comment]
Fixed in `puppeteer-extra-plugin-recaptcha@3.1.9` :)
